runtime/src/prioritization_fee_cache.rs

```diff
@@ -451,15 +485,19 @@ mod tests {
     fn sync_finalize_priority_fee_for_test(
         prioritization_fee_cache: &PrioritizationFeeCache,
         slot: Slot,
+        bank_id: BankId,
     ) {
-        prioritization_fee_cache.finalize_priority_fee(slot);
+        prioritization_fee_cache.finalize_priority_fee(slot, bank_id);
         let fee = PrioritizationFeeCache::get_prioritization_fee(
             prioritization_fee_cache.cache.clone(),
             &slot,
         );
 
         // wait till finalization is done
-        while !fee.lock().unwrap().is_finalized() {
+        while !fee
+            .get(&bank_id)
+            .map_or_else(|| false, |block_fee| block_fee.is_finalized())
```

Comment:
nit: `.map_or()` should be preferred here, because it's fine to eager-evaluate `false`.